js/controller.js

```diff
@@ -58,7 +58,9 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         "countDownFinished": false,
         "countDownCancelled": false,
         "timeToShow": 0,
-        "showing": false
+        "showing": false,
+        "delayedSetEmbedCodeEvent": false,
+        "selectedContentData": null
```

Comment:
After re-thinking it, I don't like the other possibility even more.
Let's stays as is for 2 variables. But at least reset selectedContentData after it is being consumed.